### PR TITLE
ADPUPA prepbufr python/json updates

### PR DESCRIPTION
**Pull request by PraveenKumar-NOAA**
A number of changes were made to the adpupa python script and json file: 

- TPC flags are changed for the airTemperature and virtualTemperature
- Group and type are changed for the height variable
- Variable verticalSignificance is changed to the prepbufrDataLevelCategory
- Updated input/output file names in python script and some information in json file

**Variable naming convention results:** 

  Variable MetaData/prepbufrDataLevelCategory
   ERROR: Variable 'MetaData/prepbufrDataLevelCategory' is not listed in the YAML conventions file.
  Variable QCFlags/qualityFlags
  Variable ObsValue/stationPressure
  Variable ObsValue/airTemperature
  Variable ObsValue/virtualTemperature
  Variable ObsValue/specificHumidity
  Variable ObsValue/windEastward
  Variable ObsValue/windNorthward
  Variable QualityMarker/pressure
  Variable QualityMarker/stationPressure
  Variable QualityMarker/airTemperature
  Variable QualityMarker/windEastward
  Variable QualityMarker/virtualTemperature
  Variable QualityMarker/specificHumidity
  Variable QualityMarker/windNorthward

**Final results:**
  _errors:      1_
  _warnings:    0_

A new variable MetaData/prepbufrDataLevelCategory will be added to the YAML conventions file.

### CODE REVIEW

**Review by nicholasesposito — DISMISSED** [review]
Looks good. Good job

File: ush/ioda/bufr2ioda/bufr2ioda_adpupa_prepbufr.py [hunk 1]
```diff
@@ -243,22 +242,20 @@ def bufr_to_ioda(config, logger):
...
 
     # Create the dimensions
-    dims = {
-        'Location': np.arange(0, lat.shape[0])
-    }
+    dims = {'Location': np.arange(0, lat.shape[0])}
 
     # Create IODA ObsSpace
-    iodafile = f"{cycle_type}.t{hh}z.{data_type}.nc"
+    iodafile = f"{cycle_type}.t{hh}z.{data_type}.{data_format}.{cycle}.nc"
```
Comment (CoryMartin-NOAA) [inline ush/ioda/bufr2ioda/bufr2ioda_adpupa_prepbufr.py]: Do we want cycle included in the file name? What are the others doing?

**Review by CoryMartin-NOAA — APPROVED** [review]

File: ush/ioda/bufr2ioda/bufr2ioda_adpupa_prepbufr.py (hunk 1 above)
Comment (PraveenKumar-NOAA) [reply ush/ioda/bufr2ioda/bufr2ioda_adpupa_prepbufr.py]: @CoryMartin-NOAA thanks, I removed the cycle from the filename.

**Comment by PraveenKumar-NOAA** [comment]
> Looks good. Good job

@nicholasesposito thanks, please approve the PR once you agree.

**Review by nicholasesposito — APPROVED** [review]

**Pull request merged**